cns/cnsclient/httpapi/client.go

```diff
@@ -1,6 +1,8 @@
 package httpapi
```

Comment:
although it's already in master, I think this package name is relatively ambiguous given the two client scenarios of
- External client making requests to CNS (CNI)
- Internal client making function calls to CNS (requestcontroller)

This package is used by the latter, although on first glance I would've assumed the former given that it's the one making http requests

Reply:
yes, agreed. Lets fix it in a separate PR.